Release v0.23.2 (#385)
fluxcd/image-automation-controller: CHANGELOG.md

# Changelog

## 0.23.2

**Release date:** 2022-06-08

This prerelease fixes a regression for SSH host key verification and
updates some dependencies.

> In addition, the controller dependencies where update to Kubernetes v1.24.1.

Reviewer: ```suggestion
In addition, the controller was updated to Kubernetes v1.24.1.
```
